examples/document-field-customisation/package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "@keystone-6/monorepo-example-document-field-customisation",
-  "description": "Example to demonstrate customising document field with custom components",
-  "version": "0.0.1",
+  "version": null,
   "private": true,
-  "author": "Dinesh Pandiyan <flexdinesh@gmail.com>",
```

Comment:
@flexdinesh :yellow_heart: but we don't add this field for any of the examples, that is what `git` is for